benchkit/commandattachments/tracecmd.py

```diff
@@ -0,0 +1,41 @@
+from benchkit.shell.shellasync import AsyncProcess
+from benchkit.utils.types import PathType
+from benchkit.platforms import get_current_platform
+import pathlib
+
+class trace_cmd_attachement():
+
+    def __init__(
+        self,
+        events: list[str] = []
```

Comment:
```suggestion
        events: List[str] = (),
```
where List is included from `typing`